tools/cli/example/README.md

```diff
@@ -0,0 +1,132 @@
+# dubbo-go-cli 使用示例

```

Comment:
the same as above readme. pls rename this file to readme_cn.md and add an english readme.